lib/chorde/clients/memcached.py

```diff
@@ -1031,7 +1024,7 @@ def shorten_key(self, key,
                 zpfx = self.compress_prefix
             except:
                 # Try pickling
-                key = "P#"+self.key_pickler.dumps(key,2).encode("base64").replace("\n","")
+                key = "P#"+b64encode(self.key_pickler.dumps(key,2))
```

Comment:
Don't you need to remove the `\n`?

Reply:
b64encode doesn't add them (`str.encode("base64")` did)